spec/unit/webrtc/groupCallEventHandler.spec.ts

```diff
@@ -137,6 +183,42 @@ describe('Group Call Event Handler', function() {
         mockClient.off(GroupCallEventHandlerEvent.Incoming, onIncomingGroupCall);
     });
 
+    it("handles data channel", async () => {
+        await groupCallEventHandler.start();
+
+        const dataChannelOptions = {
+            "maxPacketLifeTime": "life_time",
+            "maxRetransmits": "retransmits",
+            "ordered": "ordered",
+            "protocol": "protocol",
+        };
+
+        mockClient.emit(
+            RoomStateEvent.Events,
+            makeMockGroupCallStateEvent(
+                FAKE_ROOM_ID,
+                FAKE_GROUP_CALL_ID,
+                {
+                    "m.type": GroupCallType.Video,
+                    "m.intent": GroupCallIntent.Prompt,
+                    "dataChannelsEnabled": true,
+                    dataChannelOptions,
+                },
+            ),
+            {
+                roomId: FAKE_ROOM_ID,
+            } as unknown as RoomState,
+            null,
+        );
```

Comment:
Maybe this should be factored out into a method on MockCallMatrixClient, given that you're sending state events like this multiple times